duckduckgo/iOS
# Feature/xcode9 ui fixes

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/449543110705387
CC:

**Description**:

Update the UI for Xcode 9 - this does not include explicit fixes for iPhone X


**Steps to test this PR**:

1. Ensure the whole range of UI looks appropriate on a range of a different devices and versions of iOS
1. Confirm fixed: https://app.asana.com/0/414235014887631/431340016793877
1. Confirm fixed: https://app.asana.com/0/414235014887631/431340016793876 
1. Confirm fixed: https://app.asana.com/0/414235014887631/431340016793875
1. Confirm fixed: https://app.asana.com/0/414235014887631/431340016793874


###### Reviewer Checklist:


###### PR DRI Checklist:
